extensions/tailscale/src/status.tsx

```diff
@@ -0,0 +1,23 @@
+import { updateCommandMetadata } from "@raycast/api";
+import { getDevices, getStatus } from "./shared";
+
+export default async function Status() {
+  let subtitle: string;
+  try {
+    const data = getStatus();
+
+    // tailscale is guaranteed to be online because getStats throws if it isn't
+
+    const magicDNSSuffix = data.MagicDNSSuffix;
+    const devices = getDevices(data);
+    const activeExitNode = devices.find((d) => d.exitnode);
+
+    subtitle = `✅ Tailscale is connected on ${magicDNSSuffix}`;
+    if (activeExitNode) {
+      subtitle += ` via ${activeExitNode.name}`;
+    }
+  } catch {
+    subtitle = "❌ Tailscale is offline.";
```

Comment:
We throw a few different errors depending on why the status failed:

- `InvalidPathError` is when the `path` preference is set incorrectly
- `NotRunningError` is when Tailscale is installed but open
- `NotConnectedError` is when Tailscale is installed and open, but not connected to any tailnet

Can you update this error message to handle these three errors?

Reply:
I opted for reusing `getErrorDetails` function, so there's less code duplication